src/main/java/org/myrobotlab/service/interfaces/JoystickListener.java

```diff
@@ -1,10 +1,9 @@
 package org.myrobotlab.service.interfaces;

```

Comment:
changes to joystick interface should be done in a different pull request from service lifecycle please.

Reply:
These file only get changed updated, because I'm keeping it in sync with develop.  These are merge conflicts that have been resolved, because changes occurred on this file and were pushed into develop from another pr.